modules/layers/src/icon-layer/icon-manager.ts

```diff
@@ -417,7 +417,9 @@ export default class IconManager {
     })[]
   ): void {
     // This method is only called in the auto packing case, where _canvas is defined
-    const ctx = this._canvas!.getContext('2d') as CanvasRenderingContext2D;
+    const ctx = this._canvas!.getContext('2d', {
```

Comment:
Nit: I find the use of exclamation marks very disconcerting. Such a subtle syntax hides an override of the type system. 
```suggestion
    const ctx = this._canvas?.getContext?.('2d', {
    if (!ctx) {
      throw ...
```